### ISSUE TITLE
Image/Color Compression

### ISSUE THREAD

**Issue by shmshrbali** [minimaxir/video-to-gif-osx#2]
In your blog you state " limits the color pallete to 256 colors " Is there a way to change this setting? I couldnt find anything on gifsicle, And i dont mind the gif size being bigger for internal usage and would like the images to "look" a lot better.

**Comment by minimaxir** [comment]
Yes: the `gifsicle` command has a `--colors 256` parameter; just change the 256. I did not notice a huge benefit when increasing it, however.

**Comment by shmshrbali** [comment]
Gifsicle doesnt accept  paramter above 256 in colors.. the only other thing i found was --no-option which i couldnt get to work. Do you know how to use that?
![screen shot 2015-08-17 at 7 45 03 pm](https://cloud.githubusercontent.com/assets/2536415/9306861/a1e1fe48-4518-11e5-8e82-9ba86f4b178e.png)

**Comment by minimaxir** [comment]
Ah, that's interesting, did not know that. Would removing the `--colors 256` parameter completely work?

**Comment by shmshrbali** [comment]
According to the documentation the color options can be turned off using the command, atleast thats what it implies. Havent been able to run it, so cant confirm. If you can give it a try let me know? you can label this issue off-topic though since its nothing to do with with this repo. But i'd appreciate if you could have a look?

**Comment by minimaxir** [comment]
Ran the test case w/o the `--colors 256`. Output was larger (100KB vs. 73KB), which implies that it works to avoid color compression. However, the console yells at you:

`gifsicle: warning: too many colors, using local colormaps
     (You may want to try ‘--colors 256’.)`

**Comment by shmshrbali** [comment]
Sounds good can you tell how you got it to run for you? I'll give it a try immediately and see what kind of difference does it make in quality..

**Comment by shmshrbali** [comment]
After some experimentation i realised it was partly the imagemagick problem :- -fuzzy i removed and the rest was the dithering used which unfortunately nothing can be done about, fair about though on most cases if i remove fuzzy and dither using a different algorithm the gif's are coming out a lot smoother...
